lib/shopify_app/controller_concerns/login_protection.rb

```diff
@@ -167,6 +167,10 @@ def login_url_params(top_level:)
         query_params[:host] ||= host
       end
 
+      if params[:access_scopes].present?
+        query_params[:scope] = params[:access_scopes].join(",")
```

Comment:
Other places in this method use `||=` but I don't see a need for that here.

Reply:
Makes sense to me, if we get query args we should override them, assuming we can trust the source - is this secured in any way, like with an HMAC?

Reply:
(Discussed internally, we do not see any issue here)